services-geocoding/src/main/java/com/mapbox/api/geocoding/v6/models/V6Coordinates.java

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.api.geocoding.v6.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.mapbox.geojson.Point;
+
+import java.io.Serializable;
+
+/**
+ * Object containing coordinate parameters (lat, long) and accuracy.
+ */
+@AutoValue
+public abstract class V6Coordinates implements Serializable {
```

Comment:
Don't we need a builder for this class?

Reply:
Mostly the same as here https://github.com/mapbox/mapbox-java/pull/1550/files/eb5084da7fc95edc301986eb8fbb7b9db4a5f3cc#r1237481827